liquibase-standard/src/main/java/liquibase/analytics/AnalyticsListener.java

```diff
@@ -18,4 +18,8 @@ public interface AnalyticsListener extends Plugin {
      * @param event the event to be handled.
      */
     void handleEvent(Event event) throws Exception;
+
+    default boolean isEnabled() {
+        return true;
+    }
```

Comment:
I would not make this a default method, but instead make implementing classes provide implementations.